libsolidity/formal/SMTEncoder.h

```diff
@@ -71,6 +71,9 @@ class SMTEncoder: public ASTConstVisitor
 	static std::vector<VariableDeclaration const*> stateVariablesIncludingInheritedAndPrivate(ContractDefinition const& _contract);
 	static std::vector<VariableDeclaration const*> stateVariablesIncludingInheritedAndPrivate(FunctionDefinition const& _function);
 
+	static std::vector<VariableDeclaration const*> localVariablesIncludingModifiers(FunctionDefinition const& _function);
+	static std::vector<VariableDeclaration const*> modifiersVariables(FunctionDefinition const& _function);
```

Comment:
Can this function be protected?

Reply:
They can, I've just been adding the `static` ones as public just in case they're used from the outside somewhere, but can move into protected

Reply:
> They can, I've just been adding the `static` ones as public just in case they're used from the outside somewhere, but can move into protected

If they will be used, let's leave them public than

Reply:
Currently they're only used by the SMTEncoder itself, I made it public "just in case"